README.md

```diff
@@ -1,2 +1,45 @@
 # github-actions
 A collection of reusable and composite Github Actions
+
+## maven/build-scan-save
+A Composite action to save an unpublished Maven Build Scan®.
+
+The action uploads the Build Scan® (if present) as a workflow artifact with a random name (pattern: `scan-maven-<UUID>`).
```

Comment:
This description could be clearer: the main use case is to add a Step after each Maven execution in an _existing_ workflow file.

Reply:
Sure